content/en/tracing/code_origins/_index.md

```diff
@@ -115,12 +115,11 @@ export DD_CODE_ORIGIN_FOR_SPANS_ENABLED=true
 
 - To search for all spans that include Code Origins, use the query `@_dd.code_origin.type:*` in the [APM Trace Explorer][1].
 
-### Code preview is not visible
-
-- Ensure that all [Source Code Integration][7] setup requirements are met.
-- For transpiled Node.js applications (for example, TypeScript), make sure to generate and publish source maps with the deployed application, and to run Node.js with the [`--enable-source-maps`][10] flag. Otherwise, code previews do not work. See the Node.js [Source Code Integration][9] documentation for more details. 
-
+### Code preview is not visible, or file not found
 
+- Ensure that all [Source Code Integration][7] setup requirements are met, and that your `DD_GIT_*` tags are configured with the correct values.
+- For transpiled Node.js applications (for example, TypeScript), make sure to generate and publish source maps with the deployed application, and to run Node.js with the [`--enable-source-maps`][10] flag. Otherwise, code previews will not work. See the Node.js [Source Code Integration][9] documentation for more details.
```

Comment:
```suggestion
- For transpiled Node.js applications (for example, TypeScript), make sure to generate and publish source maps with the deployed application, and to run Node.js with the [`--enable-source-maps`][10] flag. Otherwise, code previews will not work. See the Node.js [Source Code Integration][9] documentation for more details.
```